pipeline/scheduled-pipeline-executor.groovy

```diff
@@ -0,0 +1,58 @@
+/*
+    Pipeline script for executing weekly scheduled test suites
+*/
+// Global variables section
+
+// Server to get recipe file contents
+def SSHServer = "ssh 10.245.4.89"
+
+// Recipe file path details
+def recipeFile = "RHCEPH-*.yaml"
+def recipeFileDir = "/data/site/recipe"
+
+// Default job parameters
+def buildType = "${params.buildType}" ?: "tier-1"
+def overrides = "${params.overrides}" ?: "{}"
+def tags = "${params.tags}" ?: ""
```

Comment:
Shouldn't the default value for tags include `schedule`. This being a scheduled job, I believe we should have it.

Reply:
Done